Custom ellipsoid

disberd opened the pull request:
First attempt to add support for custom rotation ellipsoid defined as function of the semi-major axis and flattening parameters.

Potentially fixes https://github.com/JuliaSpace/SatelliteToolbox.jl/issues/59

Waiting for comments from @ronisbr for the style and for other things that might need to be changed to suit the package

ronisbr commented:
Awesome work @disberd ! Thank you very much!

The first thing I want to ask is to reformat the PR according to the [BlueStyle](https://github.com/invenia/BlueStyle), which I am trying to use in this repository. The main thing to fix is the number of spaces in indentation. We use 4 spaces here.

Except for that, I have no comment. Thank you one more time for the amazing work.

disberd commented:
Thanks for the heads up about style @ronisbr, I barely never contributed to other projects before so I am still quite un-experienced with these things!

I think my latest commit should have removed all tabs, removed trailing whitespaces and split lines with more than 92 characters.
I think those were the only issues I had before wih the BlueStyle.

Let me know if anything else is needed

ronisbr commented:
Perfect! I will merge as soon as the test passes :) Thanks!
